Tests/js/views/services/plugins/assets/ez-universaldiscoverycontenttreeplugin-tests.js

```diff
@@ -136,12 +136,206 @@ YUI.add('ez-universaldiscoverycontenttreeplugin-tests', function (Y) {
         },
     });
 
+    startingLocationTests = new Y.Test.Case({
+        name: "eZ Universal Discovery Content Tree Plugin tests",
+
+        setUp: function () {
+            var that = this,
+                BrowseView = Y.Base.create('universalDiscoveryBrowseView', Y.View, [], {
+                selectContent: function (struct) {
+                    Assert.isNotUndefined(
```

Comment:
those asserts are not precise enough. In struct, you should assert that the Location is the starting Location object. The content type should be the corresponding content type and given you test with `loadContent` set to false, normally `struct.content` should be undefined.
Also, you don't check anywhere that this method is really called.